app/src/main/res/values-it/about.xml

```diff
@@ -0,0 +1,615 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">Informazioni su...</string>
+    <string name="about_close">Chiudi</string>
+    <string name="about_whoami">Io sono monerujo</string>
+    <string name="about_version">Versione %1$s (%2$d)</string>
+
+    <string name="credits_text"><![CDATA[
+        <b>Crediti</b>
+        <br/>
+        m2049r, baltsar777, anhdres, keejef,
+        rehrar, EarlOfEgo et al.
+        <br/><br/>
+        <a href="https://monerujo.io/">monerujo.io</a>
+        ]]></string>
+
+    <string name="privacy_policy"><![CDATA[
+        <h1>Politica per la Privacy</h1>
+        <p>Questa pagina ti informa sulla nostra politica riguardante la raccolta, l'utilizzo e la rivelazione di informazioni personali che riceviamo dagli utenti della nostra app (monerujo: Portafoglio Monero).
+        </p>
+        <p>Usando questa app, acconsenti alla raccolta e all'utilizzo delle informazioni in accordo con questa politica.
+        </p>
+        <h2>Dati raccolti</h2>
+        <p>Per "dato personale" si intende ogni tipo di dato grazie al quale è possibile identificare un individuo.
+        </p>
+        <p>Le chiavi e gli indirizzi pubblici di Monero vengono raccolti e processati dall'app localmente con lo scopo di processare le transazioni e vengono trasmessi all'interno della rete Monero in modo cifrato.
+        </p>
+        <p>Altri dati personali non sono raccolti dall'app.</p>
+        <p>Se usi la funzionalità (opzionale) del cambio USD/EUR, monerujo recupera il tasso di cambio attraverso le API pubbliche di kraken.com.
+        Controlla la loro politica per la privacy (in lingua inglese) su https://www.kraken.com/legal/privacy per conoscere i dettagli su come vengono raccolti i dati nelle tue richieste (specialmente la sezione "Information We Collect Automatically" - "Informazioni che raccogliamo in modo automatico).</p>
+        <p>Se utilizzi l'app per effettuare pagamenti ad indirizzi BTC, stai usando il servizio XMR.TO.
+        Controlla la loro politica per la privacy (in lingua inglese) su https://xmr.to/ per conoscere i dettagli. Monerujo invia a loro l'indirizzo di destinazione BTC e l'ammontare della transazione. Anche il tuo IP potrebbe essere raccolto.</p>
+        <h2>Permessi app</h2>
+        <ul>
+        <li>INTERNET : Connessione alla rete Monero attraverso un nodo remoto</li>
+        <li>LETTURA_UNITAMEMORIA_ESTERNA : Legge i file di portafoglio salvati all'interno del dispositivo</li>
```

Comment:
Please keep app permissions in their original form: `READ_EXTERNAL_STORAGE`

Reply:
Agreed.